pylmod/tests/test_gradebook.py

```diff
@@ -267,7 +327,21 @@ def test_get_gradebook_id(self):
         # Remove data and assert exception raised
         self._register_get_gradebook(False)
         with self.assertRaises(PyLmodUnexpectedData):
-            gradebook_id = test_gradebook.get_gradebook_id(self.GBUUID)
+            test_gradebook.get_gradebook_id(self.GBUUID)
+
+    @httpretty.activate
+    def test_get_options(self):
+        """Verify that we can get the options for a gradebook."""
+        self._register_get_options(True)
+        self._register_get_gradebook()
+        gradebook = GradeBook(self.CERT, self.URLBASE, self.GBUUID)
+        options = gradebook.get_options(gradebook_id=self.GRADEBOOK_ID)
+        self.assertIn('membershipQualifier', options)
+
+        # check for no data
+        self._register_get_options(False)
```

Comment:
To my earlier comment, you could have just reused the URI registration and changed the returned data with `del test_data['data']` and avoided needing a function or `if..then`, but this is fine as is since you already solved it this way.